cyclonedx-lib/sign_src/TemurinSignSBOM.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * # Licensed under the Apache License, Version 2.0 (the "License");
+ * # you may not use this file except in compliance with the License.
+ * # You may obtain a copy of the License at
+ * #
+ * #      https://www.apache.org/licenses/LICENSE-2.0
+ * #
+ * # Unless required by applicable law or agreed to in writing, software
+ * # distributed under the License is distributed on an "AS IS" BASIS,
+ * # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * # See the License for the specific language governing permissions and
+ * # limitations under the License.
+ * ################################################################################
+ */
+package temurin.sbom;
+
+import org.cyclonedx.BomGeneratorFactory;
+import org.cyclonedx.CycloneDxSchema;
+import org.cyclonedx.generators.json.BomJsonGenerator;
+import org.cyclonedx.model.Bom;
+import org.cyclonedx.parsers.JsonParser;
+
+import org.webpki.json.JSONAsymKeySigner;
+import org.webpki.json.JSONObjectReader;
+import org.webpki.json.JSONSignatureDecoder;
+import org.webpki.json.JSONCryptoHelper;
+import org.webpki.json.JSONAsymKeyVerifier;
+import org.webpki.json.JSONObjectWriter;
+import org.webpki.json.JSONOutputFormats;
+import org.webpki.json.JSONParser;
+import org.webpki.util.PEMDecoder;
+
+import java.io.StringReader;
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.GeneralSecurityException;
+import java.security.KeyPair;
+import java.security.PublicKey;
+import org.cyclonedx.exception.ParseException;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public final class TemurinSignSBOM {
+
+    private static boolean verbose = false;
+
+    // Create a logger for the class
+    private static final Logger LOGGER = Logger.getLogger(TemurinSignSBOM.class.getName());
+
+    private TemurinSignSBOM() {
+    }
+
+    /**
+     * Main entry.
+     * @param args Arguments for sbom operation.
+     */
+    public static void main(final String[] args) throws Exception {
+        String cmd = null;
+        String privateKeyFile = null;
+        String publicKeyFile = null;
+        String fileName = null;
+
```

Comment:
Add a new boolean success, default to false

Then in each cmd below set success to true

At end add:
```
if (!success) {
    System.exit(1);
}
```